common-test/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>com.basistech</groupId>
         <artifactId>annotated-data-model</artifactId>
-        <version>2.4.7-SNAPSHOT</version>
+        <version>2.5.0-SNAPSHOT</version>
```

Comment:
Does this require a parent-pom minor version bump?

Reply:
I don't know; does it?